File: tests/for_test.m
s = 10;
H = zeros(s);

for c = 1:s
    for r = 1:s
        H(r,c) = 1/(r+c-1);
    end
end

disp(H)

for i =  [1,2,3;4,5,6;7,8,9]
    disp(i)
end

File: tests/for_test.py
import numpy as np


s = 10
H = np.zeros((s, s))

for c in range(1, s + 1):
    for r in range(1, s + 1):
        H[r - 1:r, c - 1:c] = 1 / (r + c - 1)

print(H)

for i in np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]]):
    print(i)
